src/src/Helpers/ESPEasy_Storage.cpp

```diff
@@ -278,6 +278,10 @@ String BuildFixes()
     Settings.WiFi_sensitivity_margin = 3; // Margin in dBm on top of sensitivity.
   }
 
+  if (Settings.Build < 20113) {
+    Settings.JSONBoolWithQuotes(true); // Set JSON output to backward compatible
```

Comment:
Why not invert the boolean logic, so that a `0` is the default value?
This way you don't need this transition and also new default settings will be the old behavior.

Reply:
I'll invert the logic, and update the docs.